quill-core/src/main/scala/io/getquill/ast/AstOps.scala

```diff
@@ -0,0 +1,74 @@
+package io.getquill.ast
+
+import io.getquill.ast.{ BinaryOperation => B }
```

Comment:
I'd prefer to avoid these aliases, they make the code harder to follow

Reply:
Fixed